src/StorageSync/StorageSync/ChangeLog.md

```diff
@@ -18,9 +18,8 @@
         - Additional information about change #1
 -->
 
-## Upcoming Release
-
-## Version 0.7.1
+## Current Release
```

Comment:
Please bring this up to date - your changes should just be under Current Release

Reply:
Isn't comment is under Current Release ? 

---
In reply to: [255709474](https://github.com/Azure/azure-powershell/pull/8465#discussion_r255709474) [](ancestors = 255709474)